__tests__/sources.ts

```diff
@@ -1082,6 +1095,50 @@ query Source($id: ID!) {
   });
 });
 
+describe('query source moderation fields', () => {
+  const QUERY = `
+query Source($id: ID!) {
+  source(id: $id) {
+    id
+    name
+    image
+    public
+    moderationRequired
+    moderationPostCount
+    currentMember {
+      role
+      roleRank
+      permissions
+    }
+  }
+}
+  `;
+
+  it('should not return moderationPostCount when moderation is not required', async () => {
+    loggedUser = '1';
+    const res = await client.query(QUERY, { variables: { id: 'squad' } });
+    console.log({ res });
+    expect(res.errors).toBeFalsy();
+    expect(res.data.source.moderationRequired).toEqual(false);
+    expect(res.data.source.moderationPostCount).toBeFalsy();
+  });
+
+  it('should return moderationPostCount when user is admin', async () => {
```

Comment:
I think we are missing one more test. This is for when the user is not a moderator.

Reply:
Updated with more test cases ✅